shared-scripts.sh

```diff
@@ -502,6 +502,8 @@ function _amplifyConsoleIntegrationTests {
     retry yarn console-integration --no-cache --maxWorkers=4 --forceExit
 }
 function _integrationTest {
+    npm install -g ts-node
```

Comment:
why this line

Reply:
Because `integration_test` started to fail after removing active wait there.

Active wait was installing that here https://github.com/aws-amplify/amplify-cli/blob/0fde28a08a3d82b371cbbf858f462648376a318a/shared-scripts.sh#L649-L650 .

![image](https://github.com/aws-amplify/amplify-cli/assets/5849952/9715001c-809f-4ba5-bba4-076adc52016c)